rest_sensors.cpp

```diff
@@ -789,6 +789,42 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                 else if (rid.suffix == RConfigTempThreshold || rid.suffix == RConfigHumiThreshold)
                 {
                 }
+                else if (rid.suffix == RConfigLock)
+                {
+                    bool val = map[pi.key()].toBool();
+                    bool ok;
+                    
+                    item = sensor->item(RConfigLock);
+                    if (item)
+                    {
+                        if (val)
+                        {
+                            ok = addTaskDoorLockUnlock(task, 0x00 /*Lock*/);
+                        }
+                        else
+                        {
+                            ok = addTaskDoorLockUnlock(task, 0x01 /*UnLock*/);
+                        }
+                        if (ok)
+                        {
+                            if (item->setValue(val))
+                            {
+                                rspItemState[QString("/sensors/%1/config/lock").arg(id)] = map["lock"];
+                                rspItem["success"] = rspItemState;
+                                //if (item->lastChanged() == item->lastSet())
+                                //{
```

Comment:
Is this expected to be commented out?

Reply:
I'd suggest to remove commented out/dead code, otherwise it will sit there for ages ;)